subscription-engine/kafka-ws/Dockerfile

```diff
@@ -44,6 +44,5 @@ ENTRYPOINT ["/sbin/tini", "--"]
 
 CMD ["./bin/entryPoint.sh", "npm", "run", "kafka-ws"]
 
-HEALTHCHECK --start-period=5s --interval=30s --timeout=5s --retries=2 \
-    CMD HC=$(wget -q -O - "http://localhost:9000/health") \
-    && echo $HC && echo $HC | grep -q 'SERVER_IS_READY'
\ No newline at end of file
+HEALTHCHECK --start-period=2m --interval=30s --timeout=10s --retries=3 \
```

Comment:
Setting `start-period` to two minutes seems a little much for this service.

Reply:
You're right!